doc/source/reference/ufuncs.rst

```diff
@@ -228,45 +228,43 @@ can generate this table for your system with the code given in the Figure.
 
 .. admonition:: Figure
 
-    Code segment showing the "can cast safely" table for a 32-bit system.
+    Code segment showing the "can cast safely" table for a 64-bit system.
```

Comment:
If we want to be precise its a 64-bit system that is not windows :).

Reply:
OK, fixed (forced push).

Reply:
Maybe: `the output may depend on the system and for example differs on Windows?`, just thinking, I do not actually care too much :).

Reply:
That sounds better.

Reply:
Latest update is even more verbose. :)